lib/modules/manager/sbt/extract.spec.ts

```diff
@@ -576,7 +576,7 @@ describe('modules/manager/sbt/extract', () => {
               replaceString: 'sbt.version=1.6.0',
               versioning: 'semver',
               extractVersion: '^v(?<version>\\S+)',
-              registryUrls: ['https://repo1.maven.org/maven2'],
+              registryUrls: [],
```

Comment:
Can you run the code against a real repo? I'm not sure if the empty array is an issue for other renovate code 🤔

Reply:
I've validated this works against the reproduction repo